src/main/resources/edu/wpi/first/pathweaver/createProject.fxml

```diff
@@ -34,37 +34,40 @@
             <Insets bottom="20.0" left="40.0" right="40.0" top="10.0" />
          </padding>
          <children>
             <Label text="Project Directory" />
-            <Label text="Time Step" GridPane.rowIndex="2" />
-            <Label text="Max Velocity" GridPane.rowIndex="3" />
-            <Label text="Max Acceleration" GridPane.rowIndex="4" />
-            <Label text="Max Jerk" GridPane.rowIndex="5" />
-            <Label text="Wheel Base" GridPane.rowIndex="6" />
+            <Label text="Time Step" GridPane.rowIndex="3" />
+            <Label text="Max Velocity" GridPane.rowIndex="4" />
+            <Label text="Max Acceleration" GridPane.rowIndex="5" />
+            <Label text="Max Jerk" GridPane.rowIndex="6" />
+            <Label text="Wheel Base" GridPane.rowIndex="7" />
             <TextField fx:id="directory" editable="false" GridPane.columnIndex="1" />
             <Button fx:id="browse" mnemonicParsing="false" onAction="#browseDirectory" text="Browse" GridPane.columnIndex="2" />
             <ChoiceBox fx:id="game" prefWidth="150.0" GridPane.columnIndex="1" GridPane.rowIndex="1" />
-            <TextField fx:id="timeStep" GridPane.columnIndex="1" GridPane.rowIndex="2" />
-            <TextField fx:id="maxVelocity" GridPane.columnIndex="1" GridPane.rowIndex="3" />
-            <TextField fx:id="maxAcceleration" GridPane.columnIndex="1" GridPane.rowIndex="4" />
-            <TextField fx:id="maxJerk" GridPane.columnIndex="1" GridPane.rowIndex="5" />
-            <TextField fx:id="wheelBase" GridPane.columnIndex="1" GridPane.rowIndex="6" />
-            <Label text="Time delta between points (seconds)" GridPane.columnIndex="2" GridPane.rowIndex="2" />
-            <Label text="Maximum capable velocity of robot (ft/s)" GridPane.columnIndex="2" GridPane.rowIndex="3" />
-            <Label text="Maximum acceleration (ft/s/s)" GridPane.columnIndex="2" GridPane.rowIndex="4" />
-            <Label text="Maximum jerk (acceleration per second) (ft/s/s/s)" GridPane.columnIndex="2" GridPane.rowIndex="5" />
-            <Label text="Distance between left and right of wheel base (ft)" GridPane.columnIndex="2" GridPane.rowIndex="6" />
-            <ButtonBar prefHeight="40.0" prefWidth="200.0" GridPane.columnIndex="2" GridPane.rowIndex="7">
+            <TextField fx:id="timeStep" GridPane.columnIndex="1" GridPane.rowIndex="3" />
+            <TextField fx:id="maxVelocity" GridPane.columnIndex="1" GridPane.rowIndex="4" />
+            <TextField fx:id="maxAcceleration" GridPane.columnIndex="1" GridPane.rowIndex="5" />
+            <TextField fx:id="maxJerk" GridPane.columnIndex="1" GridPane.rowIndex="6" />
+            <TextField fx:id="wheelBase" GridPane.columnIndex="1" GridPane.rowIndex="7" />
+            <Label text="Time delta between points (seconds)" GridPane.columnIndex="2" GridPane.rowIndex="3" />
+            <Label text="Maximum velocity (length per second)" GridPane.columnIndex="2" GridPane.rowIndex="4" />
```

Comment:
"distance per second" would be more descriptive IMO.

I feel like these hints could be done as tooltips over a "help" or "?" icon to avoid screen clutter